util/libcrypto.num

```diff
@@ -4683,9 +4683,9 @@ ERR_set_error                           ?	3_0_0	EXIST::FUNCTION:
 ERR_vset_error                          ?	3_0_0	EXIST::FUNCTION:
 X509_get0_authority_issuer              ?	3_0_0	EXIST::FUNCTION:
 X509_get0_authority_serial              ?	3_0_0	EXIST::FUNCTION:
-EC_GROUP_new_by_curve_name_ex           ?	3_0_0	EXIST::FUNCTION:EC
-EC_KEY_new_ex                           ?	3_0_0	EXIST::FUNCTION:EC
-EC_KEY_new_by_curve_name_ex             ?	3_0_0	EXIST::FUNCTION:EC
+EC_GROUP_new_by_curve_name_ex           ?	3_0_0	NOEXIST::FUNCTION:EC
+EC_KEY_new_ex                           ?	3_0_0	NOEXIST::FUNCTION:EC
+EC_KEY_new_by_curve_name_ex             ?	3_0_0	NOEXIST::FUNCTION:EC
```

Comment:
Should these lines be removed completely?

Reply:
Dont they get removed when a release happens?

Reply:
I expect all of these to get cleaned up when we do the final release

Reply:
ok